EIPS/eip-3076.md

```diff
@@ -0,0 +1,292 @@
+
+The purpose of the `minimal` format was to allow for a simpler slashing protection strategy, whereby the only thing that needed to be recorded was the `slot` number of the most recently signed `block`, along with the `source` and `target` `epoch` numbers of the most recently signed attestation (while refusing to sign messages prior to these).
+
+> The advantage of the `complete` format was that it prevented false positives (meaning that it only prevented a validator from signing if it is guaranteed to incur a slashing otherwise). `complete` also supported republishing of identical messages if the `signing_root` is saved. The advantage of the `minimal` format was simplicity of implementation and succinctness of representation.
+
+However, in the process of fleshing out this EIP, we realised that we could add a couple of [conditions](#conditions) to `complete` that would allow it to cover the `minimal` case entirely.
+
+Whilst the format outlined in this specification is most similar to the old `complete`, clients who wish to follow a simpler slashing protection strategy (i.e. the old `minimal`), can do so by simply inserting a "fake" `signed_attestation` (with the correct `source_epoch` and `target_epoch` numbers), and a "fake" `signed_block` (with the correct `slot` number).
+
+Most fields in the JSON schema are strings. For fields in which it is possible to encode the value as either a string or an integer, strings were chosen. This choice was made in order to avoid issues with different languages supporting different ranges of integers (specifically JavaScript, where the `number` type is a 64-bit float). If a validator is yet to sign a block or attestation, the relevant list is simply left empty.
+
+Of the existing clients at time of writing, all clients are implementing `v4` (`"interchange_format_version": "4"`) of the specification outlined above.
+
+The difference between `v4` and `v5` is simply cosmetic -- in `v5`, the `interchange_format` field has been removed from `metadata` because there is no longer a need to cater for two different formats.
+
+The `interchange_format_version` is set to 5 because the specification went through several breaking changes during its design, incorporating feedback from implementers. 
+
+## Backwards Compatibility
+
+This specification is not backwards-compatible with versions 3 and 4 of the draft format that have been circulating amongst client teams the last few months. However, we expect clients will update to `v5` for mainnet launch.
+
+
+## Test Cases
+
+The relevant test cases can be found in [this repository](https://github.com/eth2-clients/slashing-protection-interchange-tests).
+
+
+## Implementation
+
+Implementations exist for the following clients:
+- [Teku](https://github.com/ConsenSys/teku/pull/2820)
+- [Lighthouse](https://github.com/sigp/lighthouse/pull/1544)
+- [Prysm](https://github.com/prysmaticlabs/prysm/pull/7518)
+- [Nimbus](https://github.com/status-im/nimbus-eth2/pull/1643)
```

Comment:
This is currently a bit of a hot topic among the editors at the moment, but this content feels like it would be much more appropriate for the hard fork coordination process than for the EIP.  The EIP is a technical specification, and it doesn't really care if anyone has implemented this or everyone has implemented this.  Also, when someone is reading this in 5-10 years after all of these clients are no longer maintained and the repositories/organizations have been renamed/deleted then these links will no longer be adding value.

Note: As I mentioned, this is currently a bit of a hot topic so not a blocker for draft at the moment.